src/Xamarin.Android.Build.Tasks/Microsoft.NET.Workload.Android/WorkloadManifest.targets

```diff
@@ -3,4 +3,6 @@
       Condition=" '$(TargetPlatformIdentifier)' == 'android' " />
   <Import Project="Sdk.targets" Sdk="Microsoft.Android.Sdk.BundleTool"
       Condition=" '$(AndroidPackageFormat)' == 'aab' " />
+  <Import Project="..\build\Microsoft.NET.Runtime.MonoAOTCompiler.Task.props" Sdk="Microsoft.NET.Runtime.MonoAOTCompiler.Task"
```

Comment:
this is wrong, it's will be installed on user machine

Reply:
When we get the changes here: https://github.com/dotnet/runtime/pull/48021

This might eventually be:

```xml
<Import Project="Sdk.props" Sdk="Microsoft.NET.Runtime.MonoAOTCompiler.Task" Condition=" '$(AotAssemblies)' == 'true' " />
```

_Something_ has to import this file so we get `$(MonoAOTCompilerTaskAssemblyPath)` and `<UsingTask/>` declared.

Another way for that to happen would be if there was a MonoAOT Workload that imports the file above. Is that something I should be using here instead?

If there is a MonoAOTCompiler.Workload with a `WorkloadManifest.targets` file, it could import the right file and everything will work.

Reply:
I think we could do that

@radical do you agree we should do it that way across all workload packs?